config.py

```diff
@@ -15,11 +15,18 @@ class ConfigException(Exception):
 config.read(config_filename)
 
 try:
-    required_any = config.get('Labels', 'required-labels-any')
-    required_all = config.get('Labels', 'required-labels-all')
-    banned = config.get('Labels', 'banned-labels')
-    GITHUB_TOKEN = config.get('GitHub', 'token')
-    GITHUB_SECRET = config.get('GitHub', 'secret')
+    if request.GET.get('required_any') or request.GET.get('required_all'):
```

Comment:
I don't think request is going to be defined here. Looking at the main.py file, it looks like request is imported from flask.

Reply:
Thanks. New to this Python thing.